### DISCUSSION TITLE
Automatically clean "unused" Daily Notes

### DISCUSSION THREAD

**Discussion by AoCz** [athensresearch/athens#1375] [Feature Requests]
There are a lot of days where I open Athens but don't end up writing anything. I noticed that these daily notes pages end up becoming orphan pages and unconnected nodes in Graph mode. I know you can change the graph settings to hide these orphan nodes, but It would make sense that if no changes were made to a daily note (or if the daily note is empty), Athens should automatically clean that entry. A new one can always be created for any particular date in the future. Empty daily notes pages just create clutter.

**Comment by curiousmarkus** [comment]
For me, the empty daily notes cluttering the timeline (if I scroll down the daily notes). Sometimes I try to find something in my daily notes from recent days, then it would be much better to not see the empty daily note pages.
